### PR TITLE
[SQL-DS-CACHE-45][POAE7-963]Add convert decimal to parquet fix length…

### PR DESCRIPTION
**Pull request by xieqi**
… byte array

## What changes were proposed in this pull request?

Add convertor from decimal to fix length byte array

## How was this patch tested?

No UT

### CODE REVIEW

**Comment by winningsix** [comment]
@xieqi Per required, can we add some UTs here?

**Comment by jikunshang** [comment]
> @xieqi Per required, can we add some UTs here?

I think we should add some ut for ToBytes() method.

File: oap-ape/ape-native/src/utils/ApeDecimal.h [hunk 1]
```diff
@@ -53,6 +54,54 @@ class ApeDecimal128 {
 
   const arrow::BasicDecimal128& value() const { return value_; }
 
+  int32_t ToInt32() { return static_cast<int32_t>(value_.low_bits()); }
```
Comment (jikunshang) [inline oap-ape/ape-native/src/utils/ApeDecimal.h]: Other methods in this class like `value`, `precision` are using lower case.

**Comment by xieqi** [comment]
Add UT for toBytes

File: oap-ape/ape-native/src/utils/decimalTest.cc [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  arrow::Decimal128::FromString("-9876543210.1234", &decimal, &precision, &scale);
+  ApeDecimal128 d2(decimal.high_bits(), decimal.low_bits(), precision, scale);
+  d2.toBytes(buffer.data());
+  assert(b2 == buffer);
+
+  // retsult from JAVA code
+  // BigDecimal("-12193263112635198799.878698366652").unscaledValue().toByteArray()
+  std::vector<uint8_t> b3{0xff, 0xff, 0xff, 0x66, 0x19, 0x71, 0x26, 0x43,
+                          0xad, 0x7c, 0x25, 0x58, 0x5e, 0xbf, 0x09, 0x44};
+  arrow::BasicDecimal128 d = DecimalUtil::Multiply(d1, d2, 18 + 14 + 1, 12);
```
Comment (jikunshang) [inline oap-ape/ape-native/src/utils/decimalTest.cc:52]: This line will core dump in my dev machine.

**Pull request merged**